head/firmware/main.cpp

```diff
@@ -185,6 +189,15 @@ auto main() -> int {
     head_tasks::start_tasks(can_bus_1, motor_left.motion_controller,
                             motor_left.driver, motor_right.motion_controller,
                             motor_right.driver, psd);
+    static auto timer_for_notifier =
+        freertos_timer::FreeRTOSTimer<pdMS_TO_TICKS(100)>(
+            "timer for notifier",
```

Comment:
I don't see that there's an equivalent timer in the simulator. 

There's no way you want to have a timer at 100ms in the simulator. But some way to exercise this would be nice.

Although it is completely sensible to move that to a different PR.